### PR TITLE
Select Course List

### PR DESCRIPTION
**Pull request by rusheel98**

### CODE REVIEW

File: OnlineCourseRegistration/templates/home.html [hunk 1]
```diff
@@ -31,10 +31,10 @@
             <a href="index.html" class="nav-link active">Dashboard</a>
           </li>
           <li class="nav-item px-2">
-            <a href="posts.html" class="nav-link">Students</a>
+            <a href="Students.html" class="nav-link">Students</a>
```
Comment (rusheel98) [inline OnlineCourseRegistration/templates/home.html]: updated dashboard to navigate to students tab

File: OnlineCourseRegistration/templates/home.html [hunk 2]
```diff
@@ -31,10 +31,10 @@
...
           </li>
           <li class="nav-item px-2">
-            <a href="categories.html" class="nav-link">Subjects</a>
+            <a href="course.html" class="nav-link">Subjects</a>
```
Comment (rusheel98) [inline OnlineCourseRegistration/templates/home.html]: added to visit course details

**Comment by rusheel98** [comment]
Unable to upload this file since it is a binary. Hence request you to resolve the conflict by removing the file which I uploaded(db.sqlite3)

**Comment by rusheel98** [comment]
Please note that I tested other changes and only conflict with db.sqlite3 should be resolved

**Comment by sumanthd17** [comment]
Approved Rusheel's file merge

**Pull request closed without merging**